evm/Cargo.toml

```diff
@@ -26,8 +26,8 @@ num-bigint = "0.4.3"
 once_cell = "1.13.0"
 pest = "2.1.3"
 pest_derive = "2.1.0"
-plonky2 = { version = "0.1.2", default-features = false, features = ["timing"] }
-plonky2_util = { version = "0.1.0" }
+plonky2 = { path = "../plonky2", default-features = false, features = ["timing"] }
+plonky2_util = { path = "../util" }
```

Comment:
Hm are these changes relevant to the PR?

Reply:
Oh sorry! That was so I could run the EVM tests, didn't mean to include it in the PR